app/classifier/index.cjsx

```diff
@@ -445,12 +445,32 @@ module.exports = React.createClass
     subject: null
     expertClassifier: null
     userRoles: []
+    tutorial: null
+    minicourse: null
 
   componentDidMount: ->
     @checkExpertClassifier()
     @loadClassification @props.classification
+    Tutorial.find @props.workflow
```

Comment:
I'm seeing a error thrown when clicking 'Let's go' after clicking through a tutorial on GZ:3D because the geordi prop is an empty object. The tutorial is loading before geordi finishes loading?